derat/ascenso
# Upgrade to Vuetify 2.0.17.

Upgrade to Vuetify 2, which has a bunch of breaking changes.
Here's most of what I did:

- Upgrade vuetify package from 1.5.5 to 2.0.17.
- Install @mdi/font, sass, and sass-loader packages
  (required by Vuetify 2).
- Install eslint-plugin-vuetify to add eslint rules to help
  with the upgrade.
- Update Vuetify plugin initialization.
- Make a zillion changes to margins and padding to keep
  things looking okay after changes to Vuetify's grid layout
  system.
- Modify unit test initialization to pass a Vuetify object.
  Add setUpVuetifyTesting() and newVuetifyMountOptions()
  functions to the testutil module to avoid repetitive code.
- All containers are now lazy by default, so remove 'lazy'
  from the menu in ClimbDropdown and update ClimbDropdown
  and Routes unit tests to wait for containers' contents to
  be instantiated asynchronously.
- Remove text size styling from area names in Routes view.
- Update routes page in E2E tests to accommodate v-menu now
  using divs rather than links.
- v-btn 'flat' attribute renamed to 'text'.
- v-list-tile-* renamed to v-list-item-*.
- v-expansion-panel renamed to v-expansion-panels,
  v-expansion-panel-content renamed to v-expansion-panel,
  v-expansion-panel-header added.
- v-toolbar replaced by v-app-bar, v-toolbar-side-icon
  replaced by v-app-bar-nav-icon.
- v-layout no longer has 'row' attribute; it's the default.
